tests/Controller/AnnotationControllerTest.php

```diff
@@ -82,10 +82,7 @@ public function testGetAnnotationsFromAnOtherUser($prefixUrl)
         }
 
         $this->client->request('GET', $prefixUrl . '/' . $entry->getId() . '.json');
-        $this->assertSame(200, $this->client->getResponse()->getStatusCode());
-
-        $content = json_decode($this->client->getResponse()->getContent(), true);
-        $this->assertGreaterThanOrEqual(0, $content['total']);
```

Comment:
was it really expected that for this test, testGetAnnotationsFromAnOtherUser, the request is allowed?

Reply:
I guess not